articles/00-GettingStarted.md

```diff
@@ -66,32 +84,94 @@ Which contains the following:
 
-Open the Unity Build settings, and switch to the target platform you wish to build on.
+You will need to enable the Unity Legacy XR system for each platform (`Edit -> Project Settings -> XR Settings`) as appropriate:
+
+![Unity XR Settings](/images/GettingStarted/UnityLegacyXRSettings.png)
+
+> At this time, while the new Unity XR Management system does work for some platforms, there could be unforseen issues.  the XRTK talks natively with each platform to improve performance and reduce overhead.
+> Will will be updating to support enabling XR through the new system in a future release.
+
+Once you are ready to build, open the Unity Build settings, and switch to the target platform you wish to build on.
 
 > **Note:** Depending on the platform's simulation support, you should be able to run in the editor by pressing play.
 
-Once satasfied, you can build your project by using the Mixed Reality Toolkit's Build window:
+For specific builds, please check the quickstart sections below:
+
+> Please check the FAQ for any difficulties.
+
+### Building for Windows Standalone (OpenVR/Oculus Rift-Link)
+
+When targeting Windows Standalone there should be no special setup and the Mixed Reality Toolkit will work as configured. Builds should be able to target OpenVR and Oculus Quest (via Oculus Link).
+
+Please ensure either the OpenVR or Oculus Camera and Controller providers are configured.
+
+### Building for Windows Mixed Reality (UWP)
+
+Unity allows running of UWP projects directly from the editor for simple testing.  Builds that are exported will require Visual Studio with the latest UWP API's installed to produce a build to ship and run on other WIndows 10 machines or the HoloLens.  Visual Studio also provides an independent method to publish to the Windows Store for distribution.
+
+Please ensure the `Windows Mixed Reality` XRTK package is installed and that the Windows Mixed Reality Camera and controller providers are configured.
+
+### Building for Oculus Quest / Android
+
+Building for Oculus Quest is rather unique, if you have your Oculus Quest connected to your machine via a high speed USB3 cable and have the Oculus Client running, you can run Quest project in Windows Standalone mode directly on the Quest via Oculus Link, [please see Oculus Support for further details](https://support.oculus.com/444256562873335/).  This will enable both controller and hands support for quick development.
+
+When building for Android, there is an additional requirement to install an `AndroidManaifest` for the quest to operate correctly, the XRTK provides a fast an easy way to do this by simply clicking `Mixed Reality Toolkit -> Tools -> Oculus -> Create Oculus Quest Compatible AndroidManaifest.xml` in the Unity editor menu. (**without this, your project may not start or appear in 2D**)
+
+Please ensure the `Oculus` XRTK package is installed and that the Oculus Camera and controller providers are configured.
```

Comment:
A lot of this seems repetitive per platform, couldn't we word this in a more generic way?

Reply:
Some of it is, and we can evolve it later.  However, each doc is unique in itself and we cannot guarantee readers will be willing to hope around.  GettingStarted is meant to be standalone in effect.

Experience has taught me that most readers will only read the section they are interested in and not the whole doc

Reply:
Fine but I'm personally unlikely to keep the docs up to date. I just wanna minimize the overhead of this task.

Reply:
That's where I come in 😂